seed.sh

```diff
@@ -1,21 +1,7 @@
 #!/bin/bash
```

Comment:
> We talked before about some sort of enum (#389) to avoid hard-coding collection names when using app.getConfig(). Could there be a similar principle here, with the Mongo shell commands?

Maybe... I still feel ok about how this currently is because the extra level abstraction doesn't seem like it would help too much.

> Within this file, do we even want to keep lines 20-30?

Again, maybe... but this might still be needed for updating our local config collections.